mlflow/tracking/client.py

```diff
@@ -2667,6 +2671,28 @@ def create_model_version(
             await_creation_for=await_creation_for,
         )
 
+    def copy_model_version(self, src_model_uri, dst_name) -> ModelVersion:
```

Comment:
What happens if `src_model_uri` is like `models:/model_name@alias`?

Reply:
That will work too.